src/main/java/uk/org/tombolo/importer/lac/LAQNImporter.java

```diff
@@ -30,7 +30,7 @@
 
     private static final String LAQN_PROVIDER_LABEL = "erg.kcl.ac.uk";
     private static final String LAQN_PROVIDER_NAME = "Environmental Research Group Kings College London";
-    private static final String LAQN_SUBJECT_TYPE_LABEL = "airQualityControl";
+    private static final String LAQN_SUBJECT_TYPE_LABEL = "airQualitySensor";
```

Comment:
This importer is missing a proper datasourceId and is using the subject type label for it, this is why in the tests you had to insert the subject type instead. My suggestion is to add another static variable here and use it in line 45.
`private static final String DATASOURCE_ID = "airQuality";` and in line 45
```
datasourceSpec = new DatasourceSpec(getClass(), DATASOURCE_ID, "Air Quality London",
                 "Data captured by sensors operated by London Air", dataSourceURL);
```

Now you can use the datasourceId `airQuality` in the tests and the recipes will run.